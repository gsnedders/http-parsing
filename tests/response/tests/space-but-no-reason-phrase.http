HTTP/1.1 200 
Connection: Close

Test