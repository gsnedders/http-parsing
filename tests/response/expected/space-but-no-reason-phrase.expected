{
	"HTTP": {
		"version": "1.1",
		"status": 200,
		"reason": ""
	},
	
	"headers": {
		"connection": "close"
	},
	
	"body": "Test"
}